am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestConstants.java

```diff
@@ -23,10 +26,25 @@
     public static final String OTHER_ACCESSOR_ID = "b";
     public static final Set<Permission> EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS =
         Stream.of(CREATE, READ, UPDATE).collect(toSet());
+    public static final Set<Permission> EXPLICIT_READ_PERMISSION = Stream.of(READ).collect(toSet());
+    public static final Set<Permission> EXPLICIT_CREATE_PERMISSION = Stream.of(CREATE).collect(toSet());
+    public static final Map<JsonPointer, Set<Permission>> EMPTY_ATTRIBUTE_PERMISSIONS = new ConcurrentHashMap<>();
+    public static final Map<JsonPointer, Set<Permission>> SINGLE_ATTRIBUTE_PERMISSION = new ConcurrentHashMap<>();
+
+    static {
```

Comment:
Instead of this I would prefer to have some collection factory that simplifies processionals of instantiating lists, sets and maps. Something modelled after Guava factories.